crypto3/libs/blueprint/include/nil/blueprint/zkevm_bbf/util.hpp

```diff
@@ -101,6 +101,59 @@ namespace nil {
             std::size_t memory_expansion_cost(std::size_t new_memory_byte_size, std::size_t last_memory_byte_size) {
                 return memory_cost_util(new_memory_byte_size) - memory_cost_util(last_memory_byte_size);
             }
+
+            // We define constants using the smallest possible bitlengths from nil/crypto3/multiprecision/literals.hpp
+            // Maybe it would have been better to define more bitlenghts, but this will probably all disappear, once
+            // small fields are introduced.
+            static const unsigned int two_15 = 32768;
+            static const unsigned int two_16 = 65536;
+            static const nil::crypto3::multiprecision::big_uint<64> two_32 = 4294967296;
+            static const nil::crypto3::multiprecision::big_uint<64> two_48 = 281474976710656;
+            static const nil::crypto3::multiprecision::big_uint<92> two_64 = 0x10000000000000000_big_uint92;
+            static const nil::crypto3::multiprecision::big_uint<130> two_128 = 0x100000000000000000000000000000000_big_uint130;
+            static const nil::crypto3::multiprecision::big_uint<205>
+                two_192 = 0x1000000000000000000000000000000000000000000000000_big_uint205;
+
+            template<typename T, typename V = T>
+            T chunk_sum_64(const std::vector<V> &chunks, const unsigned char chunk_idx) {
```

Comment:
I'd rather have it returning a vector with all the chunks, without specifying chunk_idx